### DISCUSSION TITLE
Is there a way to control the `TTL` of a response in `Custom filtering rules` or somewhere else?

### DISCUSSION THREAD

**Discussion by flavorgold1** [AdguardTeam/AdGuardHome#3880] [Q&A] [answered]
Unless I'm reading wrong, I don't see a way here: https://github.com/AdguardTeam/AdGuardHome/wiki/Hosts-Blocklists

**Comment by ainar-g** [comment, marked as answer]
The setting `dns.blocked_response_ttl` generally controls the TTL of all blocked or rewritten responses, if I recall correctly.  There is currently no way to control that on a per-rule basis.
